synapse/storage/databases/main/keys.py

```diff
@@ -35,7 +36,9 @@ class KeyStore(SQLBaseStore):
     """Persistence for signature verification keys"""
 
     @cached()
-    def _get_server_verify_key(self, server_name_and_key_id):
+    def _get_server_verify_key(
+        self, server_name_and_key_id: Iterable[Tuple[str, str]]
+    ) -> Dict[Tuple[str, str], FetchKeyResult]:
```

Comment:
I'm not sure about this, mainly because I don't really understand how `cached` and `cachedList` work.

I'd guess that 
- `server_name_and_key_id` (singular) should be just `Tuple[str, str]`. (It's `server_name_and_key_idS` plural below.)
- the return type ought to be just `FetchKeyResult`?

Second opinion please @matrix-org/synapse-core.

Reply:
That sounds correct.

Reply:
ok